repo/packages/J/jenkins/3/config.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": "object",
+  "properties": {
+    "jenkins": {
+      "description": "Jenkins framework configuration properties",
+      "type": "object",
+      "properties": {
+        "framework-name": {
+          "description": "The framework name to register with Mesos.",
+          "type": "string",
+          "default": "jenkins"
+        },
+        "host": {
+            "description": "The host that Jenkins agents will use to connect back to the master.",
+            "type": "string",
+            "default": "$HOST"
+        },
+        "cpus": {
+            "description": "CPU shares to allocate to each Jenkins master.",
+            "type": "number",
+            "default": 1.0,
+            "minimum": 0.1
+        },
+        "mem": {
+            "description": "Memory to allocate to each Jenkins master.",
+            "type": "number",
+            "default": 2048.0,
+            "minimum": 2048.0
+        },
+        "mesos-master": {
+            "description": "URI of this cluster's Mesos master.",
+            "type": "string",
+            "default": "zk://leader.mesos:2181/mesos"
+        },
+        "host-volume": {
+          "description": "The location of a volume on the host to be used for persisting Jenkins configuration and build data. The final location will be derived from this value plus the name set in `framework-name` (e.g. `/mnt/host_volume/jenkins`). Note that this path must be the same on all DCOS agents.",
+          "type": "string",
+          "default": "/tmp"
+        },
+        "jvm-opts": {
+            "description": "Optional arguments to pass to the JVM.",
+            "type": "string",
+            "default": "-Xms1024m -Xmx1024m"
+        },
+        "known-hosts": {
+            "description": "A space-separated list of hosts used to populate the SSH known hosts file on the Jenkins master.",
+            "type": "string",
+            "default": "github.com"
+        },
+        "hostname": {
```

Comment:
Could we make this a more generic "constraints" parameter instead? Adding `hostname` when we already have `host` seems confusing to me, especially when they do two very different things.